api/app/Policies/UserPolicy.php

```diff
@@ -92,62 +90,46 @@ public function update(User $user, User $model, array $injected)
      */
     public function delete(User $user, User $model)
     {
-        if ($user->isAdmin()) {
-            if ($user->id !== $model->id) {
-                return true;
-            } else {
-                return false; // Do not allow user to delete their own model.
-            }
-        } else {
-            return false;
-        }
+        return $user->isAbleTo('delete-any-user') && $user->id !== $model->id; // Do not allow user to delete their own model.
```

Comment:
Question about data retention here; is it not a privacy concern that we do not allow users to delete their own data? 🤔

Reply:
Hah! Fair question. But we don't really have a mechanism designed for users to delete their own data. This is to ensure we don't break the system by deleting every admin by accident.